files/class-vip-filesystem-stream-wrapper.php

```diff
@@ -650,4 +706,69 @@ protected function close_handler() {
 	protected function trim_path( $path ) {
 		return ltrim( $path, 'vip:/\\' );
 	}
+
+	/**
+	 * Validates the provided stream arguments for fopen.
+	*
+	* @since   1.0.0
+	* @access  private
+	* @param   string    $path   Path to file
+	* @param   string    $mode   fopen mode
+	*
+	* @return  bool
+	 */
+	private function validate( $path, $mode ) {
```

Comment:
If we make this a `public` method, it becomes testable with PHPUnit.
```suggestion
	public function validate( $path, $mode ) {
```